keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -611,14 +613,15 @@ that might indicate a serious problem?
 - A spike on failure events with keyword "failed spreadConstraint" in scheduler log.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
-No. This will be tested upon beta graduation.
 
 <!--
 Describe manual testing that was done and the outcomes.
 Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+Not yet, but it will be tested manually prior to upgrade.
```

Comment:
You should describe how you intend to test this. Manual steps are fine.

Reply:
Yes.